go.mod

```
@@ -47,7 +47,7 @@ require (
 	github.com/stretchr/testify v1.8.0
 	github.com/takuoki/gocase v1.0.0
 	github.com/tendermint/flutter/v2 v2.0.4
-	github.com/tendermint/spn v0.2.1-0.20220826123316-985b629a92dd
+	github.com/tendermint/spn v0.2.1-0.20220901140430-c65411cd5ab4
```

```suggestion
	github.com/tendermint/spn v0.2.1-0.20220826123316-985b629a92dd
```

New spn version has breaking changes for the CLI (should be the reason for CI failure), I plan to address those in a specific PR. Can we keep the previous version?